README.md

```diff
@@ -1 +1,80 @@
-# Capstone-project
\ No newline at end of file
+# Capstone-project
+
+#### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./MIT.md) of the design.
```

Comment:
- [OPTIONAL] Kindly this should be at the end of the file. you can move it and let make your PR look professional

Reply:
thanks I make it be at the last line of my readme file